functional-tests/src/test/resources/coh-client-template.yaml

```
@@ -16,4 +16,5 @@ spec:
     - name: CLUSTER
       value: "%%CLUSTER%%"
   imagePullSecrets:
-  - name: %%IMAGE_PULL_SECRETS%%
+%%IMAGE_PULL_SECRETS%%
```

I believe that leading white space should be here, but not certain.  
```suggestion
    %%IMAGE_PULL_SECRETS%%
```